src/routes/suggestions/find.ts

```diff
@@ -1,5 +1,8 @@
 import Joi from 'joi';
-import { ControllerGeneratorOptionsWithClient } from '../../comms/controller';
+import {
+  ControllerGeneratorOptionsWithClient,
+  ControllerGeneratorOptionsWithoutClientOrSupplier, ControllerGeneratorOptionsWithSupplier,
```

Comment:
This now contains imports that are unused. Given the change we expect, I think `ControllerGeneratorOptionsWithClientAndSupplier` should be there :).